scripts/install-binaries.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+ARG=""
+DIRECTORY="/tmp"
+
+#######################################
+# Feature flags
+#######################################
+
+# Optional custom URL prefix for all binaries
+KUBECTL_DOWNLOAD_URL="${KUBECTL_DOWNLOAD_URL:-}"
+JQ_DOWNLOAD_URL="${JQ_DOWNLOAD_URL:-}"
```

Comment:
why not default these to the public URLs if not specified and use it for the download later in the script. Then you won't need the if/else logic